docs/content/en/docs/getting-started/airgapped/airgap-steps.md

```diff
@@ -0,0 +1,36 @@
+---
+toc_hide: true
+---
+1. Download the EKS Anywhere artifacts that contain the list and locations of the EKS Anywhere dependencies. A compressed file `eks-anywhere-downloads.tar.gz` will be downloaded. You can use the `eksctl anywhere download artifacts --dry-run` command to see the list of artifacts it will download.
+   ```bash
+   eksctl anywhere download artifacts
+   ```
+   
+1. Decompress the `eks-anywhere-downloads.tar.gz` file using the following command. This will create an `eks-anywhere-downloads` folder.
+   ```bash
+   tar -xvf eks-anywhere-downloads.tar.gz
+   ```
+
+1. Download the EKS Anywhere image dependencies to the Admin machine. This command may take several minutes (10+) to complete. To monitor the progress of the command, you can run with the `-v 6` command line argument, which will show details of the images that are being pulled. Docker must be running for the following command to succeed.
```

Comment:
Looks like you are assuming they are using another machine besides the Admin machine. Should we say that somewhere in the prerequisites?

Reply:
This is the Admin machine ... I'm not sure exactly what you mean by "assuming they are using another machine". We can remove the statement about Docker if that's what's causing the confusion here

Reply:
No. You fixed the issue by saying that the Admin machine must be amd64 to do airgapped. There was a sentence that you deleted that said that the images commands "need to be run on an amd64 machine to import amd64 images to the registry mirror." It didn't say Admin machine.